aws/scenarios/vm/vm/vm.go

```diff
@@ -0,0 +1,51 @@
+package vm
+
+import (
+	"github.com/DataDog/test-infra-definitions/aws"
+	awsEc2 "github.com/DataDog/test-infra-definitions/aws/ec2/ec2"
+	"github.com/DataDog/test-infra-definitions/command"
+	"github.com/DataDog/test-infra-definitions/common/config"
+
+	"github.com/pulumi/pulumi-command/sdk/go/command/remote"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+type VM struct {
+	Context     *pulumi.Context
+	Runner      *command.Runner
+	Environment *config.CommonEnvironment
+	// TODO add file manager as soon as https://github.com/DataDog/test-infra-definitions/pull/9 is merged
+	// FileManager   *command.FileManager
+	DockerManager *command.DockerManager
```

Comment:
Having the `PackageManager` available may allow use to lazily install Docker, instantiating the `DockerManager` on demand.

Reply:
If it is still straightforward for callers, I am ok

Reply:
I just realised `command.NewDockerManager` doesn't install docker on the target, but will lazily do it on first Command - brilliant ! 

Is there any cons on keeping the `DockerManager` initialisation by default ? We could have the same question for `FileManager`: what are the cons of instantiating it even if eventually it will not be used ?

I see the advantage of having it available, if needed, but I might miss on some blockers